repos/rust-lang/gsoc.toml

```diff
@@ -0,0 +1,10 @@
+org = "rust-lang"
+name = "gsoc"
```

Comment:
I think a longer name "Google-summer-of-code" is probably better here.

Reply:
Yeah I couldn't decide what's better. Changed it to the longer version.